src/viscosity/include/antioch/kinetics_theory_viscosity.h

```diff
@@ -255,7 +255,9 @@ namespace Antioch
      _delta_star = CoeffType(1e-7) * ant_pow(Constants::light_celerity<CoeffType>(),2) * // * 1/(4*pi * eps_0) = 10^-7 * c^2
                     ant_pow(_dipole_moment * Units<CoeffType>("D").get_SI_factor(),2) /             
                      ( _LJ.depth() * Constants::Boltzmann_constant<CoeffType>() * 2 * ant_pow(_LJ.diameter() * Units<CoeffType>("ang").get_SI_factor(),3) );
-     _a = CoeffType(0.3125L) * ant_sqrt(Constants::Boltzmann_constant<CoeffType>() * _mass / Constants::pi<CoeffType>())
+           /* 5 / 16 * sqrt(pi * Boltzmann constant/pi) / (sigma^2) 
+                ~ 10^-14 float can't take 10^-28 in sqrt*/
```

Comment:
I just don't have the intuition on the scales of the `_mass` and Boltzmann's constant to immediately see what the problem is.